README.md

```diff
@@ -145,6 +145,20 @@ server.stop_and_destroy()
 server.ensure_started()
 ```
 
+CloudManager offers mode fine-grained deletion options for servers as you can choose to delete storages and
+choose what happens to their backups when deleting the server. By default the storage and their backups are
+always preserved.
+
+Following example would delete all storages attached to a server, but would keep the backups if they exist.
+
+```python
+
+from upcloud_api.storage import BackupDeletionPolicy
+
+manager.delete_server(uuid, delete_storages=True, backups=BackupDeletionPolicy.KEEP_LATEST)
```

Comment:
`BackupDeletionPolicy.KEEP_LATEST` makes me believe that only the last backup is kept, but the above sentence says it would keep all backups. So which is it?

Reply:
"keep_latest" keeps the latest backup, so change the policy here in the example, or the text above 👍

Reply:
Nice catch. Changed README.md to reflect reality. I think the example is still nice as it keeps the latest backup but deleted the storage.